src/Components/JournalPrompt/JournalPrompt.css

```diff
@@ -19,27 +12,16 @@ form {
   margin-top: 10vh;
   height: 100%;
   padding: 30px;
-  background-color: white;
+  background-color: var(--element-bkgd-color);
   width: 40vw;
 }
 
 textarea {
   width: 80%;
   height: 40vh;
+  font-size: 25px;
 }
 
 .prompt {
```

Comment:
Love the background for the whole prompt container!

One small nit-picky thing -the font size for the prompt feels too big to me! What do y'all think about making it smaller?

Reply:
yeah im into that @KatherineBlaine, it was so big that it was wrapping on a large sized screen for me!